<launch>

  <rosparam command="load" file="$(find pr)/constant.yaml"/>
  <rosparam command="load" file="$(find pr)/state.yaml"/>

  <arg name="arduino_port" value="/dev/ttyACM0"/>

  <param name="/joy_node/dev" value="/dev/input/js0"/>

  <node pkg="rosserial_python" name="serial_node"
    type="serial_node.py" args="$(arg arduino_port)"/>
  <node pkg="joy" name="joy" type="joy_node"/>

  <node pkg="pr" name="task_manager" type="pr_task_manager.py"/>
  <node pkg="pr" name="arduino_driver" type="pr_arduino_driver.py"/>

</launch>
